tools/rewriter/SourceRewriter.cpp

```diff
@@ -475,7 +475,8 @@ class FnPtrCall : public RefactoringCallback {
     auto *fpt = expr_ty->castAs<clang::PointerType>()
                     ->getPointeeType()
                     ->getAsAdjusted<clang::FunctionProtoType>();
-    fn_ptr_abi_sig[expr_ty_str] = determineAbiForProtoType(*fpt, ctxt);
+    // TODO figure out the correct arch
```

Comment:
should just be `Target` defined on line 51 right?

Reply:
Whoops, fixed.